trymito.io/styles/[slug].module.css

```diff
@@ -107,12 +183,189 @@
     border-radius: 1rem;
 }
 
-@media only screen and (min-width: 50em) {
+.post_content_container {
+    display: flex;
+    justify-content: space-between;
+    width: 100%;
+}
+
+.post_content p, .post_content li, .post_content a {
+    font-size: 1.2rem;
+}
+
+.post_content table {
+    font-size: 1rem;
+}
+
+
+/* Post CTA and TOC */
+.post_toc {
+    width: 16rem;
+    min-width: 16rem;
+    height: fit-content;
+    position: sticky;
+    top: 5rem;
+    margin-left: 2rem;
+    margin-right: 2rem;
+}
+
+.post_cta {
+    width: 16rem;
+    min-width: 16rem;
+    height: fit-content;
+    position: sticky;
+    top: 6rem;
+    background-color: #F2C94C;
```

Comment:
Want to make this a global color variable that we can reference elsewhere? It should be in globals.css